src/Microsoft.TestPlatform.PlatformAbstractions/netcore/System/ProcessHelper.cs

```diff
@@ -14,5 +16,10 @@ public string GetCurrentProcessLocation()
         {
             return Path.GetDirectoryName(Assembly.GetEntryAssembly().Location);
         }
+
+        public IntPtr GetProcessHandleById(int processId)
+        {
+            return Process.GetProcessById(processId).SafeHandle.DangerousGetHandle();
```

Comment:
What is this `DangerousGetHandle`? Is it dangerous? :P
Can you write a comment as well?

Reply:
Added comment for description.